packages/bpk-component-select/stories.js

```diff
@@ -17,11 +17,84 @@
  */
 
 import React from 'react';
+import PropTypes from 'prop-types';
 import { storiesOf } from '@storybook/react';
 import { action } from '@storybook/addon-actions';
 
 import BpkSelect from './index';
 
+const getFlagUriFromCountryCode = countryCode =>
+  `https://images.skyscnr.com/images/country/flag/header/${countryCode.toLowerCase()}.png`;
+
+const countries = [
+  { key: 0, id: 'AT', name: 'Austria', disabled: false },
+  { key: 1, id: 'BR', name: 'Brazil', disabled: false },
+  { key: 2, id: 'CN', name: 'China', disabled: false },
+  { key: 3, id: 'DJ', name: 'Djibouti', disabled: false },
+  { key: 4, id: 'EC', name: 'Ecuador', disabled: false },
+  { key: 5, id: 'GD', name: 'Grenada', disabled: false },
+  { key: 6, id: 'HT', name: 'Haiti', disabled: false },
+  { key: 7, id: 'IT', name: 'Italy', disabled: false },
+  { key: 8, id: 'US', name: 'USA', disabled: true },
+];
+class SelectWithImage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selected: 'IT',
+    };
+  }
+
+  getItemByValue = () => {
+    const { options } = this.props;
+    return val => {
+      const items = options.filter(o => o.id === val);
+      if (!items.length) throw new Error('Item does not exists');
```

Comment:
Is eslint ok with this? I didn't think it liked inline `if` statements..

Reply:
travis is happy so I'm assuming so 😄